### PR TITLE
Add nfs role in epicli

### PR DESCRIPTION
**Pull request by ks4225**
Originally created here: https://github.com/epiphany-platform/epiphany/pull/961

I believe I have addressed the points raised. Please let me know if there are still some gaps in the implementation.

Thanks,
DI team

### CODE REVIEW

File: core/src/epicli/data/common/defaults/configuration/nfs.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+kind: configuration/nfs
+title: "NFS Node Config"
+name: default
+specification:
+  nfs_defs:
+    - nfs_path: "/srv/nfs4/exports/"
+      nfs_src: "/exports"
+      nfs_export: "*(rw,sync,no_subtree_check,no_root_squash)"
+  # nfs_defs is a list of NFS definitions to be created
```
Comment (seriva) [inline core/src/epicli/data/common/defaults/configuration/nfs.yml]: Can you put the comments behind the actual lines pls? That way when using epicli init they will be included for sure.

**Comment by seriva** [comment]
You dont need to add the nfs entry in the minimal-cluster-config.yml`s. nfs is more of an edge case and the entry will still by added when people run epicli init with --all.

File: core/src/epicli/data/common/defaults/configuration/nfs.yml (hunk 1 above)
Comment (ks4225) [reply core/src/epicli/data/common/defaults/configuration/nfs.yml]: I have moved the comments now

**Comment by ks4225** [comment]
I have applied the recommended changes.

**Comment by ks4225** [comment]
Are you able to give us some insight into what will happen from now; in the feature branch and how the changes will eventually be released?

Is there anything additional you will require from our team?

Thanks,
DI team

**Pull request merged**